utility/authHelpers.js

```diff
@@ -0,0 +1,73 @@
+const { errorMessages } = require("./errors");
+const {randId} = require("./helper")
+
+// User
+const getUserFromToken = (accessToken, userStore) => {
+  const user = userStore.find((user) => user.login.username === accessToken.username);
+
+  if(!user) return {userFound: false, err: 500, message: errorMessages[500]};
+
+  return {userFound: true, user: user};
+}
+
+const getUserCart = (username, userStore) => {
+  // check if user exist in userStore
+  const user = userStore.find((user) => user.login.username === username);
```

Comment:
you dont need the brackets. 
```suggestion
  const user = userStore.find(user => user.login.username === username);
```

Reply:
This will actually crash your server if for some reason your store has no users, protect yourself with optional 
```suggestion
  const user = userStore.find((user) => user?.login?.username === username);
```